utils/env.go

```diff
@@ -0,0 +1,15 @@
+// utils/env.go: Tools to load or manage .env files
+package utils
+
+import (
+	"github.com/joho/godotenv"
+)
+
+// Load a specified .env file by specified path (relative)
+func LoadEnv(filePath string) error {
```

Comment:
I guess this is a bit redundant if we just want to import `godotenv` anywhere we need it but I do kind of like the idea of having a utils import for a bunch of stuff eventually.

Reply:
yeah i think that could be useful